cli/azd/pkg/project/framework_service.go

```diff
@@ -100,3 +101,11 @@ type CompositeFrameworkService interface {
 	FrameworkService
 	SetSource(inner FrameworkService)
 }
+
+func validatePackageOutput(packagePath string) error {
+	if entries, err := os.ReadDir(packagePath); err != nil || len(entries) == 0 {
```

Comment:
Separate error messages? if `err != nil`, we should at least surface the underlying error